zappa/core.py

```diff
@@ -1120,6 +1126,7 @@ def create_lambda_function(
         layers=None,
         concurrency=None,
         docker_image_uri=None,
+        architecture=None,
```

Comment:
Since we have already initialized Zappa with the architecture attribute, can we remove the architecture parameter here?